README.md

```diff
@@ -34,6 +36,21 @@ These status codes and messages appear in transactions created by the DEX (as pa
 - `t1,${takerChain},${takerOrderId},${makerOrderCount}: Orders taken`
 - `t2,${makerChain},${makerOrderId},${takerOrderId}: Order made`
 
+- Taker is the account/wallet who takes the trade from someone else (the maker).
+- Maker is responsible for initialising the trade.
+- All Market orders makes up takers.
+- Pending limit orders in orderbook makes up makers.
```

Comment:
`Pending limit orders in the order book will be makers.` The word `makes` here is confusing.